src/ballot-forms.php

```diff
@@ -1,36 +1,33 @@
 <?php
 require_once 'includes/classes/db-connector.php';
 require_once 'includes/session-handler.php';
```

Comment:
Use proper directory separator. Please refer to the file-utils.php within the classes folder.